int8 id # the ID of the init slot being notified
---
